dev/TabView/InteractionTests/TabViewTests.cs

```diff
@@ -244,6 +244,27 @@ public void AddButtonTest()
             }
         }
 
+        [TestMethod]
+        public void KeyboardTest()
```

Comment:
>KeyboardTest [](start = 20, length = 12)

May want to add a negative test or two that verify that Ctrl+Alt+F4 and Ctrl+Shift+F4 don't close the tab - the current code gets it right, but this gets messed up often so tests to protect against that regressing seem good.

Reply:
It actually didn't work with my previous code, but now with the accelerator it does. I don't think additional testing is necessary since the accelerator should (and does) scope input properly.